crates/bevy_pbr/src/render/pbr_fragment.wgsl

```diff
@@ -418,6 +418,43 @@ fn pbr_input_from_standard_material(
 #endif  // VERTEX_TANGENTS
 #endif  // VERTEX_UVS
 
+        // Take anisotropy into account.
+#ifdef VERTEX_TANGENTS
+#ifdef STANDARD_MATERIAL_ANISOTROPY
+
+        var anisotropy_strength = pbr_bindings::material.anisotropy_strength;
+        var anisotropy_direction = pbr_bindings::material.anisotropy_rotation;
+
+        // Adjust based on the anisotropy map if there is one.
+        if ((pbr_bindings::material.flags & pbr_types::STANDARD_MATERIAL_FLAGS_ANISOTROPY_TEXTURE_BIT) != 0u) {
+            let anisotropy_texel = pbr_functions::sample_texture(
+                pbr_bindings::anisotropy_texture,
+                pbr_bindings::anisotropy_sampler,
+#ifdef STANDARD_MATERIAL_ANISOTROPY_UV_B
+                uv_b,
+#else   // STANDARD_MATERIAL_ANISOTROPY_UV_B
+                uv,
+#endif  // STANDARD_MATERIAL_ANISOTROPY_UV_B
+                bias,
+            ).rgb;
+
+            let anisotropy_direction_from_texture = normalize(anisotropy_texel.rg * 2.0 - 1.0);
+            anisotropy_direction =
+                mat2x2(anisotropy_direction.xy, anisotropy_direction.yx * vec2(-1.0, 1.0)) *
```

Comment:
This looks like it's doing a common 2D rotation but I'll admit I can't remember it's not obvious to me that it's correct, could you add a comment?

Alternatively, things like `let TBN = fns::calculate_tbn_mikktspace(mesh.world_normal, mesh.world_tangent);` inspire confidence so perhaps a helper fn could be introduced.